openstack/identity/v3/projects/requests.go

```diff
@@ -50,3 +50,36 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 		return ProjectPage{pagination.LinkedPageBase{PageResult: r}}
 	})
 }
+
+// GetOptsBuilder allows extensions to add additional parameters to
+// the Get request.
+type GetOptsBuilder interface {
+	ToProjectGetQuery() (string, error)
+}
+
+// GetOpts allows you to modify the details included in the Get request.
+type GetOpts struct{}
```

Comment:
There _are_, but not implemented. I forget why I didn't implement them. There's a note in the Issue about it:

> #164 does not support the request parameters as described in the API reference and here, though Get has been implemented with GetOpts (and extendible) support for the future.